src/App.js

```diff
@@ -30,8 +36,93 @@ class App extends React.Component {
   };
 
   // Insert form callback functions handleChange and handleSubmit here
+  handleChange(event) {
+    this.setState({ input: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    let userInput = this.state.input;
+
+    if (userInput === "") {
```

Comment:
```suggestion
    if (!userInput) {
```

empty string is falsy